Xamarin.Forms.Maps.Android/MapRenderer.cs

```diff
@@ -51,6 +51,24 @@ public override SizeRequest GetDesiredSize(int widthConstraint, int heightConstr
 		{
 			return new SizeRequest(new Size(Context.ToPixels(40), Context.ToPixels(40)));
 		}
+		
+		public virtual void OnMapReady(GoogleMap map)
```

Comment:
I much prefer needed data passed to methods rather than fetched from properties. The more idempotent the function, the better.